tools/kagami/src/main.rs

```diff
@@ -504,8 +504,8 @@ mod config {
         impl<T: Write> RunArgs<T> for Args {
             fn run(self, writer: &mut BufWriter<T>) -> Outcome {
                 let config = ConfigurationProxy {
-                    torii_api_url: Some(SmallStr::from_str(DEFAULT_API_URL)),
-                    torii_telemetry_url: Some(SmallStr::from_str(DEFAULT_TORII_TELEMETRY_URL)),
+                    torii_api_url: Some(format!("http://{DEFAULT_API_ADDR}").parse().unwrap()),
```

Comment:
Consider using either `expect` or `?` preferably.

Reply:
You must communicate to the user that you have asserted the correctness and infallibility of the `parse()` call.